latest/Dockerfile.amd64

```diff
@@ -10,11 +10,11 @@ LABEL org.opencontainers.image.documentation="https://github.com/owncloud-ci/tra
 ARG BUILD_VERSION
 
 # renovate: datasource=pypi depName=transifex-client
-ENV TRANSIFEX_VERSION="${BUILD_VERSION:-0.14.4}"
+ENV TRANSIFEX_VERSION="${BUILD_VERSION:-1.6.4}"
 
 RUN apt-get update && \
     apt-get install -y git-core perl gettext liblocale-po-perl liblocale-gettext-perl python3 python3-pip && \
-    pip3 install -U "transifex-client==$TRANSIFEX_VERSION" && \
+    wget "https://github.com/transifex/cli/releases/download/v${TRANSIFEX_VERSION}/tx-linux-amd64.tar.gz" -O - | tar xz -C /usr/local/bin tx && \
```

Comment:
Nitpicking, but I would also like to use the curl syntax from https://github.com/owncloud-ci/alpine/blob/2b1422afccc7ed4488d920216ff4e0ea1da7779e/latest/Dockerfile.amd64#L41-L43 to keep it equal